main_console.js

```diff
@@ -0,0 +1,126 @@
+var startBoard = [
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9]
+];
+var board = [];
+var player1 = "X";
+var player2 = "O";
+var messages = {
+  welcome: "Welcome to Tic Tac Toe!",
+  player1First: "Player One goes first! You are 'X'. Choose an unoccupied position from the board!",
+  player2First: "Player Two goes next! You are 'O' Choose an unoccupied position from the board!",
+  player1Next: "Player One's turn again!",
+  player2Next: "Player Two's turn again!",
+  wrongMove: "That move is not allowed, try again."
+}
+var message = "";
+var player = "";
+var moves = 0;
+
+var printBoard = function(board) {
+  var line = [];
+  for (i = 0; i < board.length; i++) {
+    line[i] = board[i][0].toString() + board[i][1].toString() + board[i][2].toString();
+  }
+  return (line[0] + '\n' + line[1] + '\n' + line[2]);
+};
+
+var startGame = function() {
+  startBoard = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9]
+  ]
+  board = startBoard.slice(0);
+  player = player1;
+  moves = 0;
+  message = messages.welcome;
+  console.log(message);
```

Comment:
You could remove 38 and change to `console.log(messages.welcome)` to make things a bit more concise